packages/enzyme-matchers/src/assertions/toHaveState.js

```diff
@@ -30,11 +30,10 @@ function toHaveState(
   ) {
     return {
       pass: state.hasOwnProperty(stateKey),
-      message: `Expected <${wrapperName}> to have any value for the prop "${stateKey}"`,
-      negatedMessage: `Expected <${wrapperName}> not to receive the prop "${stateKey}"`,
+      message: `Expected the state for <${wrapperName}> to contain a value for key "${stateKey}", but it did not.`,
```

Comment:
again, i think this should be `to contain the key "stateKey"`. Good catches all around!

Reply:
Updated